packages/models/src/consumer.ts

```diff
@@ -1,6 +1,11 @@
 import { prop } from '@typegoose/typegoose';
+import { ObjectId } from 'mongoose';
 
 export class Consumer {
+    _id: ObjectId;
+
+    id: string;
+
```

Comment:
Why do we need two ids? By the way what do you think about adding a name field? It would be easier to debug and operate with it

Reply:
mongoose automatically provides fields `_id` and `id`
`id` is a string representation of `_id`. Without this type declarations typescript will alarm that `_id` and `id` don't exist

Reply:
Consumer already has `name` field

Reply:
oh yeah, haven't seen it